plugins/digitalocean/test-fixtures/config.yaml

```
@@ -0,0 +1,1188 @@
+    no-header: false
+serverless:
+  connect:
+    apihost: ""
+    auth: ""
+  deploy:
+    apihost: ""
+    auth: ""
+    build-env: ""
+    env: ""
+    exclude: ""
+    include: ""
+    incremental: false
+    insecure: false
+    remote-build: false
+    verbose-build: false
+    verbose-zip: false
+    yarn: false
+  get-metadata:
+    env: ""
+    exclude: ""
+    include: ""
+    project-reader: false
+  init:
+    language: javascript
+    overwrite: false
+  status:
+    credentials: false
+    languages: false
+    version: false
+  undeploy:
+    all: false
+    apihost: ""
+    auth: ""
+    force: false
+    packages: false
+    triggers: false
+  watch:
+    apihost: ""
+    auth: ""
+    build-env: ""
+    env: ""
+    exclude: ""
+    include: ""
+    insecure: false
+    remote-build: false
+    verbose-build: false
+    verbose-zip: false
+    yarn: false
+size:
+  list:
+    format: ""
+    no-header: false
+snapshot:
+  delete:
+    force: false
+    format: ""
+    no-header: false
+  get:
+    format: ""
+    no-header: false
+  list:
+    format: ""
+    no-header: false
+    region: ""
+    resource: ""
+spec:
+  get:
+    deployment: ""
+    format: yaml
+  validate:
+    schema-only: false
+sql-mode:
+  get:
+    format: ""
+    no-header: false
+ssh-key:
+  create:
+    format: ""
+    no-header: false
+    public-key: ""
+  delete:
+    force: false
+  get:
+    format: ""
+    no-header: false
+  import:
+    format: ""
+    no-header: false
+    public-key-file: ""
+  list:
+    format: ""
+    no-header: false
+  update:
+    format: ""
+    key-name: ""
+    no-header: false
+tag:
+  apply:
+    resource: []
+  delete:
+    force: false
+  get:
+    format: ""
+    no-header: false
+  list:
+    format: ""
+    no-header: false
+  remove:
+    resource: []
+triggers:
+  list:
+    format: ""
+    function: ""
+    no-header: false
+user:
+  create:
+    mysql-auth-plugin: ""
+  delete:
+    force: false
+  get:
+    format: ""
+    no-header: false
+  list:
+    format: ""
+    no-header: false
+volume:
+  create:
+    desc: ""
+    format: ""
+    fs-label: ""
+    fs-type: ""
+    no-header: false
+    region: ""
+    size: 4TiB
+    snapshot: ""
+    tag: []
+  delete:
+    force: false
+  get:
+    format: ""
+    no-header: false
+  list:
+    format: ""
+    no-header: false
+    region: ""
+  snapshot:
+    format: ""
+    no-header: false
+    snapshot-desc: ""
+    snapshot-name: ""
+    tag: []
+volume-action:
+  attach:
+    wait: false
+  detach:
+    wait: false
+  get:
+    action-id: 0
+    format: ""
+    no-header: false
+  list:
+    format: ""
+    no-header: false
+  resize:
+    region: ""
+    size: 0
+    wait: false
+vpcs:
+  create:
+    description: ""
+    ip-range: ""
+    name: ""
+    region: ""
+  delete:
+    force: false
+  get:
+    format: ""
+    no-header: false
+  list:
+    format: ""
+    no-header: false
+  update:
+    default: false
+    description: ""
+    name: ""
```

While I like it that we're using a realistic config file here, having all the zero-values in there does hurt readability, so I'm slightly leaning in favor of trimming it down to remove the zero-values:
```suggestion
access-token: dop_v1_tr33mpd5m8q9t3ncisqbceydi8dd2n60pl1yiycg97z25fkqffp8j6ycjexample
config: ${HOME}/Library/Application Support/doctl/config.yaml
context: default
output: text
```